qiwis.py

```diff
@@ -362,24 +363,26 @@ def destroyApp(self, name: str):
         logger.info("Destroyed the app %s", name)
 
     def updateFrames(self, name: str):
-        """Updates the frames of an app.
+        """Updates frames of the given app.
         
         Args:
-            name: A name of the app to update its frames.
+            name: The app name to update its frames.
         """
         app = self._apps[name]
         info = self.appInfos[name]
         orgFrames = {
             wrapperWidget.widget(): wrapperWidget
             for wrapperWidget in self._wrapperWidgets[name]
         }
-        newFrames = app.frames()
+        newFrames = {frame: title for title, frame in app.frames()}

```

Comment:
Now both are dictionary, so it looks quite confusing.
How about `wrapperWidgets` and `frameTitles`, leaving the sets?

Reply:
Could you explain more? I didn't understand it..

Reply:
For those `dict`s, how about renaming `orgFrames` and `newFrames` e.g. `wrapperWidgets` and `frameTitles`, respectively?

Reply:
Oh, I see! It looks more legible, so I will adopt it.